### ISSUE TITLE
add a parameter for secondary source source version update

### ISSUE THREAD

**Issue by venkatkriish** [awslabs/aws-codebuild-jenkins-plugin#81]
We are trying to update the source version of secondary source in codebuild but unable to find any option to do this through plugin is there any option to do this please update

**Comment by leobaran-aws** [comment]
Thanks for reaching out--we're working on adding this functionality. I'll keep you updated.

**Comment by venkatkriish** [comment]
Thanks @leobaran-aws

**Comment by leobaran-aws** [comment]
Implemented in 20c4f3aea2f758eada4d7f42be6be0ede00a1719. You are now able to override secondary source versions (as well as secondary source and artifacts) in the `Build Configuration` section. Just supply the information in JSON format, like so

```
[
    {
        "sourceIdentifier": "source1",
        "sourceVersion": "branch1"
    },
    {
        "sourceIdentifier": "source2",
        "sourceVersion": "branch2"
    }
]
```

**Comment by venkatkriish** [comment]
Thanks 👍
